NEWS

```diff
@@ -8,6 +8,9 @@ issues]].
 ** New features
 ** Bug fixes
 ** Braille table improvements
+- Improved back-translation and made yaml tests for pt-g1.utb,
```

Comment:
A come to think of it, the NEWS file is for human consumption so maybe it should say which language this is, i.e. Portuguese grade 1 or whatever it is. You can put the actual file name in brackets.

Reply:
I would leave out the file name completely. We usually also don't mention test improvements.